modules/machineset-yaml-rhv.adoc

```diff
@@ -59,8 +59,10 @@ spec:
           type: <workload_type> <19>
           userDataSecret:
             name: worker-user-data
+            auto_pinning_policy: <auto_pinning_policy> <20>
+            hugepages: <hugepages> <21>
```

Comment:
both fields needs to be one indentation less, and I think we should have the `userDataSecret` last so basically change it to:

```yaml
          type: <workload_type> <19>
          auto_pinning_policy: <auto_pinning_policy> <20>
          hugepages: <hugepages> <21>
          affinityGroupsNames:
            - compute <22>
          userDataSecret:
            name: worker-user-data
```